lib/matobjplist.gi

```diff
@@ -1292,3 +1294,9 @@ InstallMethod( NewCompanionMatrix,
     return ll;
   end );
 
+# tell method selection that objects know their BaseDomain
+# (this should be in the first section above, but that causes
+# various "method matches more than one declaration" messages)
+InstallTrueMethod(HasBaseDomain, IsPlistVectorRep);
+InstallTrueMethod(HasBaseDomain, IsPlistMatrixRep);
```

Comment:
The "method matches more than one declaration" messages are caused by "hidden implications", i.e. the same as issue https://github.com/gap-system/gap/issues/1649. :-(

That's quite annoying...